src/main/java/backend/resource/TurboIssue.java

```diff
@@ -334,17 +329,35 @@ public IssueMetadata getMetadata() {
     public void setMetadata(IssueMetadata metadata) {
         this.metadata = metadata;
     }
+
     public Optional<LocalDateTime> getMarkedReadAt() {
         return markedReadAt;
     }
+
     public void setMarkedReadAt(Optional<LocalDateTime> markedReadAt) {
         this.markedReadAt = markedReadAt;
     }
+
     public boolean isCurrentlyRead() {
-        return isCurrentlyRead;
+        if (!getMarkedReadAt().isPresent()) {
+            return false;
+        }
+        if (getUpdatedAt() == null) {
```

Comment:
Not likely since Github will set `updatedAt` to `createdAt` the first time. But I will assign it to the `createdAt` in the constructor then, for safety.